.docker/docker-compose.yml

```diff
@@ -1,5 +1,12 @@
 version: '3'
 services:
+  rabbitmq:
```

Comment:
Should also add the rabbitmq container to docker-compose.dev.yml and ocker-compose.local.yml files